Week2Project/used git commands.txt

```diff
@@ -0,0 +1,44 @@
+git init
+git remote add qa-reskill-repo https://github.com/Vesnamc/qa-reskill-repo.git
+git add VesnaMijicCase.txt
+git config --global user.email "vessnamc@gmail.com"
+git commit -m "first commit"
+git branch -M main
+git push -u qa-reskill-repo main
+
+git remote set-url qa-reskill-repo https://github.com/PetkovicStefan/QAReskill.git
+git pull https://github.com/PetkovicStefan/QAReskill.git --allow-unrelated-histories
+git checkout -b branch-vesna-mijic
+git commit -help
+git add VesnaMijicFile1.txt
+git status
+git commit -o VesnaMijicFile1.txt
+
+git push -u qa-reskill-repo branch-vesna-mijic
+
+
+touch .gitignore
+echo 'VesnaMijicFile1 - Copy.txt' >> .gitignore
+git check-ignore *
+
+git status
+git add VesnaMijicFile1.txt
+git commit -m "add text"
+git pull
+(merge from InteliJ)
+git commit -m "resolved merge"
+git branch -u qa-reskill-repo/branch-vesna-mijic
+git push
+
+git fetch qa-reskill-repo master:master
```

Comment:
E ovde ste svi pravili greskicu, trebali ste da se prebacite na master granu prvo, povuces sve izmene sa nje, posle toga odradis git merge qa-reskill-repo/branch-vesna-mijic, on ce ti reci da imas konflikte, to resis, posle toga git add, commit i push -u origin master i na taj nacin si odradila samo promene na master grani a ne i na svojoj